modules/core/src/v2/coins/eos.ts

```diff
@@ -284,4 +307,81 @@ export class Eos extends BaseCoin {
     };
     return { halfSigned: txParams };
   }
+
+  /**
+   * Deserialize a transaction
+   * @param txHex
+   * @param transactionHeaders
+   */
+  deserializeTransaction({ txHex, transactionHeaders }: ExplainTransactionOptions): Bluebird<object> {
+    return co(function *() {
+      const eosClientConfig = { chainId: this.getChainId(), transactionHeaders };
+      const eosClient = new EosJs(eosClientConfig);
+
+      // Get tx base values
+      const eosTxStruct = eosClient.fc.structs.transaction;
+      const serializedTxBuffer = Buffer.from(txHex, 'hex');
+      const transaction = Fcbuffer.fromBuffer(eosTxStruct, serializedTxBuffer);
```

Comment:
What is the type of this var? Can we do better than `any`?

Reply:
yes, I added an interface for `DeserializedEosTransaction`